### DISCUSSION TITLE
GiftCard da scalare

### DISCUSSION THREAD

**Discussion by dmigliacci** [fattureincloud/api#455] [Q&A] [answered]
Buongiorno,
avrei bisogno di inserire uno sconto numerico (non percentuale) sul totale ivato della fattura.

C'è un modo per farlo?

Avrei trovato il campo **amount_due_discount** ma non trovo molte informazioni

https://developers.fattureincloud.it/docs/guides/invoice-creation#seven-step-seven-advanced-options

Grazie per l'aiuto

**Comment by valmoz** [comment]
Buongiorno @dmigliacci 
consigliamo di inserire un item con valore negativo, come indicato in [questa guida](https://help.fattureincloud.it/help/articolo/196-inserisci-sconto-documento).

Grazie
Mauro

**Reply by dmigliacci** [reply]
Ciao Mauro,
però poi con che iva lavora quell'item? 0%?

![Screenshot 2024-06-13 alle 11 55 24](https://github.com/fattureincloud/api/assets/43272717/fe26390e-0d48-4044-813e-0d1f423e2ed7)

Non c'è un modo per farlo sul totale finale? Come fosse lo sconto d'arrotondamento al ristorante? 😄

**Reply by emajo** [reply, marked as answer]
Ciao @dmigliacci consiglio di guardare [questa discussione](https://github.com/fattureincloud/api/discussions/299).

**Reply by dmigliacci** [reply]
Mi sembra proprio l'indicazione che cercavo @emajo 
Procedo con _amount_due_discount_ 

Per ora riterrei chiusa la discussione, top!
